src/pages/Archive/Archive.js

```diff
@@ -0,0 +1,26 @@
+import { Header, NavBar,ArchiveNote} from "../../components/index"
+import {useArchive} from "../../context"
+import "../commonPage.css"
+import "./archive.css"
+
+
+export const Archive = () => {
+    const {archiveNotes} = useArchive()
+    return(
+        <>
+         <main className="main-page">
+            <Header/>
+            <NavBar/>
+             <div className="page-content">
+                 <div className="archive-notes">
+                     {
+                         archiveNotes && archiveNotes.map(archiveNote => (
+                            <ArchiveNote key={archiveNote.id} archiveNote={archiveNote}/>
```

Comment:
I think the archiveNotes condition check is redundant here. If no archvies are present anyways list won't print anything. Please do check this before merging.

Reply:
It is only for the safer side if anything not present then it won't render